molgenis-data-rest/src/main/java/org/molgenis/data/rest/exception/IncompatibleValueTypeException.java

```diff
@@ -0,0 +1,41 @@
+package org.molgenis.data.rest.exception;
+
+import org.molgenis.data.meta.model.Attribute;
+
+import java.text.MessageFormat;
+
+import static java.util.Objects.requireNonNull;
+import static org.molgenis.data.i18n.LanguageServiceHolder.getLanguageService;
+
+public class IncompatibleValueTypeException extends RestApiException
+{
+	private final static String ERROR_CODE = "R01";
```

Comment:
have you considered naming the field after the code instead of naming all fields ERROR_CODE?